src/app/project/project.component.ts

```diff
@@ -233,31 +232,31 @@ export class ProjectComponent implements OnInit {
       [],
       {
         relativeTo: this.route,
-        queryParams: { tab: this.selectedMainTabKey },
+        queryParams: {tab: this.selectedMainTabKey},
         queryParamsHandling: 'merge'
       });
   }
 
   private fetchProjectEntities(projectData, entityType: string, params?): Observable<PagedData<any>> {
     const projectUrl = projectData._links['self']['href'];
     return this.ingestService.get(`${projectUrl}/${entityType}`, {params}).pipe(
-        map(data => data as ListResult<any>),
-        map(data => {
-              // TODO always get the content for now
-              return {
-                data: (data && data._embedded ? data._embedded[entityType] : []).map(resource => resource['content']),
-                page: data.page
-              };
-            }
-        ));
+      map(data => data as ListResult<any>),
+      map(data => {
+          // TODO always get the content for now
```

Comment:
seems what it's doing. is this still a todo?

Reply:
I only autoformat it, I don't know (or can't remember what this TODO) is for, will just keep it.

Reply:
This loc isn't part of the changes made for this PR.

@jacobwindsor I can see your name against this bit of code, would you advise removing the to-do?